node/cmd/guardiand/keygen.go

```diff
@@ -0,0 +1,49 @@
+package guardiand
+
+import (
+	"crypto/ecdsa"
+	"crypto/rand"
+	"log"
+
+	"github.com/certusone/wormhole/node/pkg/common"
+
+	ethcrypto "github.com/ethereum/go-ethereum/crypto"
+	"github.com/spf13/cobra"
+)
+
+var keyDescription *string
+var blockType *string
+
+func init() {
+	keyDescription = KeygenCmd.Flags().String("desc", "", "Human-readable key description (optional)")
+	blockType = KeygenCmd.Flags().String("block-type", "", "block type of armored file (optional)")
+}
+
+var KeygenCmd = &cobra.Command{
+	Use:   "keygen [KEYFILE]",
+	Short: "Create guardian key at the specified path",
+	Run:   runKeygen,
+	Args:  cobra.ExactArgs(1),
+}
+
+func runKeygen(cmd *cobra.Command, args []string) {
+	common.LockMemory()
```

Comment:
This LockMemory call has been there since the beginning, I just moved the code to a new file. That being said, this command runs as a short lived, stand alone instance of guardiand, so I don't think this call will impact the long running guardian, or even badgerdb.